stdlib/src/array.rs

```diff
@@ -1424,12 +1424,13 @@ mod array {
         }
     }
 
-    #[derive(FromArgs)]
+    #[derive(FromArgs, PyTraverse)]
     struct ReconstructorArgs {
         #[pyarg(positional)]
         arraytype: PyTypeRef,
         #[pyarg(positional)]
         typecode: PyStrRef,
+        #[notrace]
```

Comment:
```suggestion
        #[pytraverse = false]
```
I feel like this attr can be enhanced. Instead of using `PyTypeRef`, we can use the underlying type `PyRef<PyType>`. Because only `PyRef<T>` and `PyObjectRef` are ref-counted types, recognizing them and automatically marking `#[pytraverse = true]` maybe useful.
(Though still Arg* needs to be manually marked)

Reply:
Types that can't contain PyRef/PyObjectRef don't need to be trace as them can't form a cycle by their own, this should reduce a lot of work, on the other hand wouldn't `#[pytraverse = false]` be a litte too verbose?


> I feel like this attr can be enhanced. Instead of using `PyTypeRef`, we can use the underlying type `PyRef<PyType>`. Because only `PyRef<T>` and `PyObjectRef` are ref-counted types, recognizing them and automatically marking `#[pytraverse = true]` maybe useful. (Though still Arg* needs to be manually marked)

Reply:
Cancelling about other parts, but still vote for `#[pytraverse = false]` or `#[pytraverse(false)]`.
keeping using same name `pytraverse` or `Traverse` will be helpful to recognize what it is.

Reply:
> Cancelling about other parts, but still vote for `#[pytraverse = false]` or `#[pytraverse(false)]`. keeping using same name `pytraverse` or `Traverse` will be helpful to recognize what it is.

How about `#[pytraverse(no)]` or `#[pytraverse(skip)]` on fields of struct? Feeling `#[pytraverse = false]` maybe not easily extendable for future